src/coreclr/ToolBox/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -699,29 +702,27 @@ void MethodContext::repCompileMethod(CORINFO_METHOD_INFO* info, unsigned* flags)
     info->locals = SpmiRecordsHelper::Restore_CORINFO_SIG_INFO(value.info.locals, CompileMethod, SigInstHandleMap);
 
     *flags             = (unsigned)value.flags;
-    DEBUG_REP(dmpCompileMethod(0, value));
 }
 
 void MethodContext::recGetMethodClass(CORINFO_METHOD_HANDLE methodHandle, CORINFO_CLASS_HANDLE classHandle)
 {
     if (GetMethodClass == nullptr)
         GetMethodClass = new LightWeightMap<DWORDLONG, DWORDLONG>();
 
-    GetMethodClass->Add(CastHandle(methodHandle), CastHandle(classHandle));
-    DEBUG_REC(dmpGetMethodClass(CastHandle(methodHandle), CastHandle(classHandle)));
+    DWORDLONG key = CastHandle(methodHandle);
+    GetMethodClass->Add(key, CastHandle(classHandle));
```

Comment:
Yeah, I thought about doing the same for `value` that I did for `key` -- creating separate variables for them in all cases to avoid multiple `CastHandle` calls. Decided to stop with the `key` changes for now.

Reply:
well, since I had to re-submit my changes, I went ahead and created `value` assignments everywhere to reduce the number of casts, and regularize the code even more.